overlord/overlord.go

```diff
@@ -269,6 +283,12 @@ func (o *Overlord) Loop() {
 			o.stateEng.Ensure()
 			select {
 			case <-o.loopTomb.Dying():
+				if o.udevMon != nil {
```

Comment:
Could this move under the "Stop()" function of overlord? It seems that the place where we also manage the state engine stop handling so it seems natural to do the udevMon stop there too (unless I miss something of course :)